.github/workflows/ci.yml

```diff
@@ -2,9 +2,10 @@
 name: CI
 
 on:
+  workflow_dispatch:
```

Comment:
Why not, if that happens in the future at least we'll be able to trigger a run manually 👍